contracts/registries/IPAssetRegistry.sol

```diff
@@ -79,6 +79,34 @@ contract IPAssetRegistry is IIPAssetRegistry, IPAccountRegistry, Governable {
         emit IPRegistered(id, block.chainid, tokenContract, tokenId, name, uri, registrationDate);
     }
 
+    function registerCrossChain(uint256 chainId, address tokenContract, uint256 tokenId) external returns (address id) {
+        if (block.chainid == chainId) {
```

Comment:
why revert here?  shouldn't it just work as intended?

Reply:
Logic changed, has removed the revert.